dev/com.ibm.websphere.appserver.features/visibility/public/javaeeClient-8.0/com.ibm.websphere.appserver.javaeeClient-8.0.feature

```diff
@@ -14,12 +14,12 @@ IBM-API-Package: \
   com.ibm.websphere.appserver.jndiClient-1.0, \
   com.ibm.websphere.appserver.jpa-2.2, \
   com.ibm.websphere.appserver.beanValidation-2.0, \
-  com.ibm.websphere.appserver.jaxwsClient-2.2, \
+  com.ibm.websphere.appserver.jaxwsClient-2.3, \
   com.ibm.websphere.appserver.j2eeManagementClient-1.1, \
   com.ibm.websphere.appserver.jsonp-1.1, \
   com.ibm.websphere.appserver.jsonb-1.0, \
   com.ibm.websphere.appserver.javaMail-1.6, \
-  com.ibm.websphere.appserver.jaxb-2.2, \
```

Comment:
This change is not intended to be delivered, I'm using it as a means to test the jaxwsClient-2.3 feature in a PB. I will remove it before merging these changes.